src/docs/sphinx/quickstart.rst

```diff
@@ -267,28 +267,268 @@ must be specified using either:
 
 ``python3 scripts/uberenv/uberenv.py --spack-config-dir=scripts/spack/configs/linux_ubuntu_18 --prefix=../path/to/install``
 
-Preparing OSX for Serac Installation
+Building Serac dependencies on MacOS
 ------------------------------------
 
 .. warning::
-   These instructions are in development and are only one example of how to handle this.
+   These instructions are in development.
 
-.. warning::
-   OSX MPI may throw a popup that causes all MPI runs to fail.
+Meeting base dependency requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One way to install the required depedencies is with the MacPorts package manager.
+Suggested ports to install:
+
+* clang-12 (or another version as you prefer)
+* openmpi-clang12 (matching the version of clang you installed)
+* gcc12 (or another version; this is needed only for a fortran compiler)
+* cmake
+* autoconf
+* automake
+* graphviz
+* pkgconfig
+* xorg-libX11
 
-Install required compilers and MPI via `homebrew <https://brew.sh/>`_:
+Activate the particular package versions with MacPorts:
 
 .. code-block:: bash
 
-  $ brew install gcc@8
-  $ brew install llvm@11
-  $ brew install mpich
+   $ sudo port select clang mp-clang-12
+   $ sudo port select mpi openmpi-clang-fortran
```

Comment:
should be

`sudo port select mpi openmpi-clang12-fortran`

instead of 

`sudo port select mpi openmpi-clang-fortran`

Reply:
Thanks! updated.